src/tests/Interop/CommandLine/CommandLineTest.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Reflection;
+using TestLibrary;
+using Xunit;
+
+namespace CommandLineTest
+{
+    class Program
```

Comment:
I would prefer this to follow the `[Fact]` pattern we are moving towards.

Removing the namespace, making the class `public`, renaming to `CommandLineTest` and then renaming `int Main()` as follows should be all that is needed. Also, removing the return as this will all be generated for you.

```csharp
[Fact]
public void Validate_CommandLineArgsSet()
```

Reply:
This test does not really need to be under src/tests. It can be under libraries (and using remote executor). It is a test for `Environment.GetCommandLineArgs()` library API after all.

Reply:
I considered that under xunit we may have much less control of the actual command line of the process. Can xunit be run in-process by Visual Studio?

Reply:
@AaronRobinsonMSFT What's your thought about the expected value here? I'm afraid the only thing we can test is not empty, which looks not enough.

Reply:
> @AaronRobinsonMSFT What's your thought about the expected value here? I'm afraid the only thing we can test is not empty, which looks not enough.

In general, I agree. However, the libraries teams have traditionally made an effort to call every API even if it is only to verify is always throws or return nothing. It does make for some odd tests, but is helpful in ensuring the API works enough to invoke. More to the point, I think it is already done https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.Extensions/tests/System/Environment.GetCommandLineArgs.cs.

Once I update CoreShim to be more useful we can create the more robust suite I think we all want.